web/app.fluidity.money/app/util/chainUtils/ethereum/transaction.ts

```diff
@@ -368,6 +403,33 @@ export const getUserStakingDeposits = async (
   }
 };
 
+// Call Static Stake tokens - For Error Checking
+export const testMakeStakingDeposit = async (
+  signer: Signer,
+  stakingAbi: ContractInterface,
+  stakingAddr: string,
+  lockDurationSeconds: BN,
+  usdcAmt: BN,
+  fusdcAmt: BN,
+  wethAmt: BN,
+  slippage: BN,
+  maxTimestamp: BN
+) => {
+  const stakingContract = getContract(stakingAbi, stakingAddr, signer);
+
+  if (!stakingContract)
+    throw new Error(`Could not instantiate Staking Contract at ${stakingAddr}`);
+
+  // call deposit
+  return await stakingContract.callStatic.deposit(
+    lockDurationSeconds.toString(),
```

Comment:
If BigNumber was used internally exclusively a string conversion wouldn't be needed since it can work with BigNumberish